plugins/CoreHome/templates/_dataTableCell.twig

```diff
@@ -5,9 +5,6 @@
     <a rel="noreferrer noopener"
        target="_blank"
        href='{% if row.getMetadata('url')|slice(0,4) not in ['http','ftp:'] %}http://{% endif %}{{ row.getMetadata('url')|rawSafeDecoded }}'>
-    {% if not row.getMetadata('logo') %}
```

Comment:
Having this icon outside of the label `span` caused some really weird sizing bugs. The problem is that the label `span` has `width:inherit;`, so it copies the width of the parent element. The parent element gains a fixed width from the resizing logic. But this width can't be used for the label `span` if there is an additional icon outside, as it makes the column wider that it should be.